src/github.com/getlantern/flashlight/client/balancer.go

```diff
@@ -37,7 +38,10 @@ func (client *Client) initBalancer(cfg *ClientConfig) (*balancer.Balancer, error
 			log.Debugf("Adding chained server: %v", s.Addr)
 			dialers = append(dialers, dialer)
 		} else {
-			log.Errorf("Unable to configure chained server. Received error: %v", err)
+			elog.Log(err, errlog.WithOp("configure"), errlog.WithProxy(&errlog.ProxyingInfo{
```

Comment:
My knowledge of Go idioms is not very extense, you might have a cleaner idea of what a interface-base solution might look like. My impression is that subtyping with dumb objects Go might prefer embedding.